### ISSUE TITLE
Ask: TCP fully supported?

### ISSUE THREAD

**Issue by Zhaojp-Frank** [openucx/ucx#1810]
Hi, 
I'm trying to evaluate UCX, a few quick questions about HW/SW dependency as as to understand whether it fits my use case:
1. if I have a normal 10GbE NIC, do most of UCX API/lib still work? I don't expect RDMA
2. in such normal TCP case, UCX uses kernel TCP stack or any use-space (sounds doesn't depend on DPDK)
I'm assuming it's typically for MPI App, but not depend on MPI, right?

**Comment by miked-mellanox** [comment]
yep, should work because UCX does support ROCE (tcp also in plans)
UCX is independant on MPI and provides network transport abstraction which is suitable for many app cases.
